app/components/telegram_widget/telegram_widget.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="TelegramWidget">
+  <script
+    async
+    src="https://telegram.org/js/telegram-widget.js?12"
+    data-telegram-login="<%= ENV['TELEGRAM_BOT_USERNAME'] %>"
+    data-size="large"
+    data-auth-url="https://<%= ENV['APPLICATION_HOSTNAME'] %>/onboarding/telegram-auth"
+    data-request-access="write">
+  </script>
+</div>
```

Comment:
I considered not creating a component for this and just rendering this in the onboarding instructions template, but decided that we might want to use it on other pages if we support logging in and letting users manage settings, or something similar.